_projects/2022/100348375/100348375.Rmd

```diff
@@ -0,0 +1,981 @@
+two years.
+
+2. *perc_increase_label*, to add a "+" to the previous variable in case the 
+value is positive. 
+
+3. *increase2*, with three categories to differentiate positive increases below 
+40%, positive increases above 40% and negative.
+
+4. *industry_labels*, keeps the *industry* label only in case the percentage 
+increase is greater than 40%.
+
+The second data frame contains the market share data for the three technology 
+industries starting in 2012 and an additional column with a 1. This data frame 
+will serve to draw the three dots that indicate that these data are slightly 
+different from the rest.
+
+```{r}
+# First data frame:
+
+alternative_plot_data <- data %>%
+  ungroup() %>% 
+  mutate(perc_increase = round(increase/market_share_2000*100, 1),
+         perc_increase_label = case_when(perc_increase > 0 ~ 
+                                           paste0("+", perc_increase),
+                       T ~ as.character(perc_increase)),
+         increase2 = case_when(perc_increase > mean(perc_increase) ~ 
+                                 "high_increase",
+                               T ~ increase2),
+         industry_labels = case_when(increase2 != "high_increase" ~ "",
+                              T ~ industry),
+         "Early 2000s" = 1,
+         Now = 2) %>% 
+  arrange(desc(increase2))
+
+head(alternative_plot_data)
+```
+
+```{r}
+# Second data frame:
+
+data.frame.points <- data %>%
+  ungroup() %>% 
+  filter(industry == "Smartphones" | industry == "Social networks" 
+         | industry == "Cellphone service") %>% 
+  select(market_share_2000, industry) %>% 
+  mutate(x = 1)
+
+head(data.frame.points)
+```
+
+### **Create the static graph**
+
+### DATA, COORDINATES AND AXES
+
+Parallel coordinate plots can be easily created with the ggplot2 extension 
+"ggally", as explained [here](https://r-graph-gallery.com/parallel-plot.html). 
+However, in this case I have had to do it with a combination of geom_segment and 
+"geom_vline" because the ggally functions don't support interactivity with either 
+ggplotly or ggiraph. The function used from the ggiraph package is called 
+"geom_segment_interactive".
+With "geom_vline" I have created the two vertical and parallel lines where the 
+values of the market share in 2012 and currently are represented. 
+Subsequently, with the geom_segment function I have created the lines that join 
+the values between one date and another.
+In the geom_segment aesthetic I have specified that I want the color and alpha 
+to be associated with the previously created variable *increase2* that 
+differentiates between positive, negative and positive values above 40% 
+increment. To set the values of both aesthetics I have used the 
+"scale_color_manual" and "scale_alpha_manual" functions.
+
+```{r fig.showtext=TRUE, fig.height=7, fig.width=6}
+
+library(ggiraph)
+library(grid)
+library(ggrepel)
+sysfonts::font_add_google("Catamaran", family = "catamaran")
```

Comment:
I believe that this should be called only once. Please remove other calls from below (unless you find any issue).

Reply:
(The same for the Noto Serif font you use in other chunks)